README.md

```diff
@@ -158,14 +158,16 @@ For an example, see the [Blackrock Neurotech CerePlex driver](https://github.com
 Dependencies:
 
     git submodule update --init
-    pip install -r requirements.txt
+    pip install .
     ./setup.sh all
     # or
     make all
 
 To build and install in development mode:
 
-    pip install -e .
+    git submodule update --init
+    pip install -e .[dev]
```

Comment:
I think you need

```suggestion
    pip install -e ".[dev]" 
```
otherwise the brackets are interpreted (I get `zsh: no matches found: .[dev]`)